Nef_3/include/CGAL/normal_vector_newell_3.h

```diff
@@ -70,6 +112,14 @@ void normal_vector_newell_3( IC first, IC last, Vector& n )
     // three.
 {
     CGAL_assertion( !CGAL::is_empty_range( first, last));
+
+    if(internal_nef::is_triangle_3(first)) {
+      n = orthogonal_vector(*first,*(std::next(first)),*(std__next(first,2)));
```

Comment:
You should try to compile `std__next` probably does not compile.
```suggestion
      n = orthogonal_vector(*first,*(std::next(first)),*(std::next(first,2)));
```